src/ts/app.vue

```diff
@@ -1,160 +1,29 @@
-                    <b-form-group id="grpA" label="First argument" label-for="input-a">
-                        <b-form-input id="input-a" v-model.number="input.a" type="number" required placeholder="first argument"></b-form-input>
-                    </b-form-group>
-
-                    <b-form-group id="grpB" label="Second argument" label-for="input-b">
-                        <b-form-input id="input-b" v-model.number="input.b" type="number" required placeholder="second argument"></b-form-input>
-                    </b-form-group>
-
-                    <b-form-group id="grpOp" label="Operation" label-for="input-op">
-                        <b-form-select id="input-op" v-model="input.operation" :options="operations" required></b-form-select>
-                    </b-form-group>
-
-                    <b-button id="calcButton" type="submit" variant="primary" :disabled="!canCalc">Calculate</b-button>
-                    <b-button id="resetButton" type="reset" variant="danger">Reset</b-button>
-                </b-form>
-
-                <hr />
-
-                <b-row>
-                    <b-col id="resultCol">
-                        <div v-if="result !== null" id="resultDiv">
-                            Result: <strong class="result">{{ result }}</strong>
-                        </div>
-                        <div v-if="message.length > 0" id="messageDiv">
-                            <strong class="message">{{ message }}</strong>
-                        </div>
-                    </b-col>
-
-                    <b-col>
-                        <foot-notes></foot-notes>
-                    </b-col>
-                </b-row>
-            </b-col>
-        </b-row>
-    </b-container>
+    <main-view></main-view>
 </template>
 
 <style lang="less" scoped>
-    h2 {
-        margin: 0.5em 0;
-    }
-
-    /*
-        Could be done with Vue too. But Vue is for the logic and not for the cosmetic, as here.
-        In Vue the result and message is set, because their visibility depends on the logic.
-        Hence this case here should be handled by CSS.
-    */
-    @media (max-width: 250px) {
-        .img-logo {
-            display: none !important;
-        }
-    }
-
-    .result {
-        color: #0094ff;
-    }
-
-    .message {
-        color: #fe4444;
-    }
 </style>
 
 <script lang="ts">
     import { Vue, Component } from "vue-property-decorator";
-    import * as calc          from "@svc/calculator";
 
-    import Footer from "@cmp/footer.vue";
+    import Loading      from "@cmp/loading.vue";
+    import LoadingError from "@cmp/loading-error.vue";
+
+    const MainView = () => ({
+        // Webpack will prefetch this for us, so no need for /* webpackPreload: true */ 
+        component: import(/* webpackChunkName: "main" */ "@view/main.vue") as any,     // https://github.com/vuejs/vue-class-component/issues/323#issuecomment-479834166
+        loading  : Loading,
+        error    : LoadingError,
+        delay    : 0
```

Comment:
Ah, not good. So it blinks on fast internets (when read from cache). Some delay is good. Maybe that's why vue has chosen 200ms as default. Keep it.